src/parser/command.rs

```diff
@@ -134,7 +139,9 @@ pub fn parse_defwaveform<'a>(input: ParserInput<'a>) -> ParserResult<'a, Instruc
         token!(RParenthesis),
     ))(input)?;
     let parameters = parameters.unwrap_or_default();
-    let (input, sample_rate) = token!(Float(v))(input)?;
+    // NOTE: this cast will cause unexpected behavior for very large integers that cannot be accurately cast to f64.
```

Comment:
Would it be worth checking for that and outputting an `Err`?

Reply:
Took a second to grok what `nom` wanted here, but this has now been fixed

Reply:
Nice!